### DISCUSSION TITLE
Monocle users stats

### DISCUSSION THREAD

**Discussion by andre-lima** [change-metrics/monocle#1040] [Ideas]
Hello.
I want to suggest that you keep (if possible) a list of companies where Monocle is used.

This information is important to convince more users to join in.

**Comment by morucci** [comment]
Hi Andre,

Thanks for opening that discussion. As of now we don't have such a list.

In order to get more insight about Monocle users we could add a note, at the beginning of the README.md file, to invite people to open a PR and add tan entry into a predefined list (somewhere in the README.md). The list could be a table with such info:

| company name / project name | main usage (board, metrics, other) |public instance (url)/private instance  |
|-------------------------------------------------------|---------------------------------------------------|-----------------------------------------------|
|  changemetrics            |  project demo instance                     |    https://demo.changemetrics.io                         |

What do you think about that proposal ? Also @TristanCacqueray any thoughts ?

**Reply by andre-lima** [reply]
Sounds good to start. Once you get a few, it could also be added to the https://changemetrics.io/ page.

**Reply by morucci** [reply]
Here is the related PR to add the table https://github.com/change-metrics/monocle/pull/1046 do not hesitate to comment.
